.github/workflows/automatus-ubuntu2204.yaml

```
@@ -0,0 +1,177 @@
+          name: output.json
+        # continue even if the file is unavailable that
+        # means there are no changes detected by CTF in the previous job
+        continue-on-error: true
+      - name: Test if there are no content changes
+        if: ${{ steps.get_ctf_output.outcome == 'success' }}
+        run: echo "CTF_OUTPUT_SIZE=$(stat --printf="%s" output.json)" >> $GITHUB_OUTPUT
+        id: ctf
+      - name: Print changes to content detected if any
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        run: cat output.json
+      - name: Generate id_rsa key
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        run: ssh-keygen -N '' -t rsa -f ~/.ssh/id_rsa
+      - name: Build test suite container
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        run: podman build --build-arg "CLIENT_PUBLIC_KEY=$(cat ~/.ssh/id_rsa.pub)" -t ssg_test_suite -f test_suite-ubt22
+        working-directory: ./Dockerfiles
+      - name: Get oscap-ssh
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        run: |
+          wget https://raw.githubusercontent.com/OpenSCAP/openscap/maint-1.3/utils/oscap-ssh
+          sudo chmod 755 oscap-ssh
+          sudo mv -v oscap-ssh /usr/local/bin
+          sudo chown root:root /usr/local/bin/oscap-ssh
+          rm -f oscap-ssh
+      - name: Get rule ids to be tested
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        id: rules
+        uses: notiz-dev/github-action-json-property@v0.2.0
+        with:
+          path: 'output.json'
+          prop_path: 'rules'
+      - name: Get product attribute
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        id: product
+        uses: notiz-dev/github-action-json-property@v0.2.0
+        with:
+          path: 'output.json'
+          prop_path: 'product'
+      - name: Get bash attribute
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        id: bash
+        uses: notiz-dev/github-action-json-property@v0.2.0
+        with:
+          path: 'output.json'
+          prop_path: 'bash'
+      - name: Get ansible attribute
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        id: ansible
+        uses: notiz-dev/github-action-json-property@v0.2.0
+        with:
+          path: 'output.json'
+          prop_path: 'ansible'
+      - uses: actions/download-artifact@v4
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        with:
+          name: ${{ env.DATASTREAM }}
+      - name: Run tests in a container - Bash
+        if: ${{steps.bash.outputs.prop == 'True' && steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        run: tests/test_rule_in_container.sh --no-remove-machine-only --dontclean --logdir logs_bash --remediate-using bash --name ssg_test_suite --datastream $DATASTREAM ${{join(fromJSON(steps.rules.outputs.prop))}}
+        env:
+          ADDITIONAL_TEST_OPTIONS: "--duplicate-templates --remove-fips-certified --product rhel8"
```

```suggestion
          ADDITIONAL_TEST_OPTIONS: "--duplicate-templates --remove-fips-certified"
```
This is a workaround for Fedora and RHEL